src/main/java/by/itechart/api/controller/UserController.java

```diff
@@ -20,33 +22,32 @@
     private final UserServiceImpl userService;
 
     @GetMapping("/all")
+    @RolesAllowed("ROLE_ADMIN")
     public ResponseEntity<List<UserDTO>> getAllUsers() {
         return new ResponseEntity<>(userService.findAll(), HttpStatus.OK);
     }
 
-    /**
-     * status FOUND here is just for test. It will be changed later
-     * this method will get a authentication parameter later, but for now there will be no params
-     *
-     * @return current user
-     */
+    @RolesAllowed({"ROLE_ADMIN", "ROLE_USER"})
     @GetMapping("/current")
-    public ResponseEntity<UserDTO> getCurrentUser() {
-        return new ResponseEntity<>(HttpStatus.FOUND);
+    public ResponseEntity<UserDTO> getCurrentUser(Authentication authentication) {
+        return new ResponseEntity<>(userService.getCurrentUser(authentication), HttpStatus.FOUND);
     }
 
+    @RolesAllowed({"ROLE_ADMIN", "ROLE_USER"})
```

Comment:
Since you don't have any checks any user can edit any other users. That's a security breach